Update support_01_about_dmp.md (#77)
DHP-stottepakke/pages: pages/dmp_supporting_info/support_01_about_dmp.md

## Question-specific guidance

### DMP title
Make a title for the data management plan which makes it clear which research project it relates to, if a project has multiple plans, then add information about which part or package of the project this specific plan is for.

> Eg. "Data Management Plan for AMAZING - A really fantastic project, 2020-2024"

Reviewer: ```suggestion
E.g. "Data Management Plan for AMAZING - A really fantastic project, 2020-2024"
```

> Eg. "Data Management Plan for AMAZING - A really fantastic project, WP3 the complex data collection, 2020-2024"

Reviewer: ```suggestion
E.g. "Data Management Plan for AMAZING - A really fantastic project, WP3 the complex data collection, 2020-2024"
```

### Contributor(s)
List all person contributing to both the plan and the data management in the project.

> Name is written Given-name Sur-name; Eg. Niels Henrik Abel (eller skal det være invertert???)

Reviewer: I don't think it really matters?

Author: om det høstes kan det ha noe å si og det er greit å være konsekvent så det er mulig å håndtere det videre.. men jeg vet ikke hva som er loven - men lettere å få alle til å skrive rett fram enn til å invertere - er jo ingen som skjønner det i DvNO

> Name is written Given-name Sur-name; Eg. Niels Henrik Abel (eller skal det være invertert???)

Reviewer: ```suggestion
Name is written Given-name Family-name; Eg. Niels Henrik Abel.
```

> Institutional email addresses is proffered

Reviewer: ```suggestion
Institutional email addresses as contact email are preferred.
```

> [ORCIDs](https://orcid.org/) are important to correctly identify researchers, if contributors do not have an ORCID they can create one for free.

Reviewer: ```suggestion
[ORCIDs](https://orcid.org/) are recommended to unambiguously identify researchers. If contributors do not have an ORCID they can create one for free.
```

Affiliations are added at an institutional level and harvested from the [ROR registry of research organizations.](https://ror.org/)

> Roles should be added to specify their contribution, examples of roles are: 

Reviewer: ```suggestion
Roles should be added to specify their contribution (following the [DataCite contributor type](https://datacite-metadata-schema.readthedocs.io/en/4.5/appendices/appendix-1/contributorType/ "2024-09-09") definitions). Examples of roles are: 
```

*Contact Person 
*Data Collector 
*Data Manager  
*Project Leader 
*Project Manager 
*Project Member 
*Researcher 
*Supervisor 
*Work Package Leader
*Related Person or Other can be used for others who contribute in more general terms.

### This DMP is related to other DMP(s)

> If the project has multiple plans and one "master plan" this field is used to show relations. Create the master plan first, and then sub-plans for the different parts with different practices for how data is organized.

Reviewer: ```suggestion
If there is several data management plans within a project, this field is used to show relations. Create the "master plan" first, and then sub-plans for the different parts of the project with different practices for how data is organized.
```
